### PR TITLE
HOCS-3112 Regression runs only on main

### PR DESCRIPTION
**Pull request by kiranhomeoffice**
Minor change to drone.yml to restrict automated regression runs to main branch.

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -3,7 +3,10 @@ kind: pipeline
 type: kubernetes
 name: DCU
 trigger:
+  branch:
+    - master
   event:
+    - pull_request
```
Comment (zuzak) [inline .drone.yml:9]: Please don't do anything expensive or scary on a `pull_request` -- anyone with a GitHub account can open a pull request against this repo.

File: .drone.yml [hunk 2]
```diff
@@ -3,7 +3,10 @@ kind: pipeline
 type: kubernetes
 name: DCU
 trigger:
+  branch:
+    - master
```
Comment (zuzak) [inline .drone.yml:7]: These need to be `main`; the `master` branch doesn't exist.

File: .drone.yml [hunk 3]
```diff
@@ -89,7 +92,10 @@ kind: pipeline
 type: kubernetes
 name: DECSRegression
 trigger:
+  branch:
+    - master
```
Comment (LiamMacP) [inline .drone.yml:96]:
```suggestion
    - main
```

**Review by zuzak — APPROVED** [review]
This change looks fine.

It's probably worth reexamining this Drone file at some point: there's a few duplicate steps that don't need to be (for example, we're pushing to Quay on every pipeline, instead of doing it once and then triggering the subsequent pipelines).

**Comment by LiamMacP** [comment]
> This change looks fine.
> 
> It's probably worth reexamining this Drone file at some point: there's a few duplicate steps that don't need to be (for example, we're pushing to Quay on every pipeline, instead of doing it once and then triggering the subsequent pipelines).

Adding to this - we can push the reporting all up at once instead of per pipeline.

**Pull request merged**